pkg/ipamd/ipamd.go

```diff
@@ -632,6 +632,14 @@ func (c *IPAMContext) configureIPRulesForPods() error {
 			log.Warnf("UpdateRuleListBySrc in nodeInit() failed for IP %s: %v", info.IP, err)
 		}
 	}
+
+	// Program IP rules for external service CIDRs and cleanup stale rules.
+	extServiceCIDRs := c.networkClient.GetExternalServiceCIDRs()
+	err = c.networkClient.UpdateExternalIpRules(extServiceCIDRs)
+	if err != nil {
+		log.Warnf("UpdateExternalServiceIpRules in nodeInit() failed")
```

Comment:
Function name in the log message and the actual function name don't match. I think the one in log message is more clear and intuitive.